### ISSUE TITLE
Contactlist/log one firstname doubling

### ISSUE THREAD

**Issue by biopsin** [SimpleMobileTools/Simple-Dialer#570]
Hi, updated to 5.17.5 and there is a issue in Contacts and Call history in the dialer.
I have mostly contacts with only one firstname and all of these are now doubled in list like "Home Home" but some contacts with two names in the firstname does not double.

**Comment by W1zardK1ng** [comment]
Yes have the exact same issue with version 5.17.5, if the contact is having only first name its doubled, but contacts with fist name and Surname displays correctly.
Also there Favorites are empty now, but in contacts app have added Favorite contacts.

**Comment by tibbi** [comment]
retry with 5.17.8

**Comment by biopsin** [comment]
@tibbi sorry forgot to mention Im on 5.17.8.
